Source/WebCore/PAL/pal/PALSwift/CryptoKitShim.swift

```diff
@@ -92,31 +96,30 @@ public final class AesGcm: NSObject {
                 )
             }
             sealedBox.ciphertext.copyBytes(to: cipherText, count: Int(plainTextSize))
-            sealedBox.tag.copyBytes(to: cipherText + Int(plainTextSize), count: sealedBox.tag.count)
+            sealedBox.tag.copyBytes(
+                to: cipherText + Int(plainTextSize), count: sealedBox.tag.count)
```

Comment:
No bounds check here, so we need a wrapper that expresses the bounds check.

Reply:
https://github.com/WebKit/WebKit/pull/25602#discussion_r1531120724